File: sgpi.client/src/app/interfaces/cotacao-item.ts
import { ItemCatalogo } from './item-catalogo';
import { Fornecedor } from './fornecedor';

export interface CotacaoItem {
  id: number;
  itemCatalogoId: number;
  itemCatalogo?: ItemCatalogo;
  fornecedorId: number;
  fornecedor?: Fornecedor;
  precoUnitario: number;
  dataCotacao: Date;
  linkProduto?: string;
}

File: Sgpi.Server/Core/Entities/CotacaoItem.cs
using System;

namespace SGPI.Core.Entities
{
    public class CotacaoItem
    {
        public int Id { get; set; }
        public int ItemCatalogoId { get; set; }
        public ItemCatalogo? ItemCatalogo { get; set; }
        public int FornecedorId { get; set; }
        public Fornecedor? Fornecedor { get; set; }
        public decimal PrecoUnitario { get; set; }
        public DateTime DataCotacao { get; set; }
        public string? LinkProduto { get; set; }
    }
}
